core/src/main/java/org/apache/carbondata/core/datastore/chunk/store/impl/LocalDictDimensionDataChunkStore.java

```diff
@@ -84,6 +84,9 @@ public void fillVector(int[] invertedIndex, int[] invertedIndexReverse, byte[] d
     vector = ColumnarVectorWrapperDirectFactory
         .getDirectVectorWrapperFactory(vectorInfo, vector, invertedIndex, nullBitset,
             vectorInfo.deletedRows, false, false);
+    if (dictionaryVector.getIntsSize() < rowsNum) {
```

Comment:
add a comment about when dictionaryVector.getIntsSize() < rowsNum, I think it is only during array of string / varchar with local dictionary .

Reply:
done